project/plugins.sbt

```diff
@@ -1 +1,6 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.6")
\ No newline at end of file
+resolvers += Resolver.sonatypeRepo("releases")
+addSbtPlugin("com.codacy" % "codacy-sbt-plugin" % "17.0.2")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.15")
+
+libraryDependencies += "com.lihaoyi" %% "ujson" % "0.7.5" // to parse Json in the build.sbt file
```

Comment:
as said, can we try to use `sjson`?

Reply:
Please no more JSON libraries in the code base

Reply:
Refactored using the already provided sjsonnew library.